Bigtable/src/ResumableStream.php

```diff
@@ -116,6 +121,17 @@ public function __construct(
         $this->callOptions['retrySettings'] = [
             'retriesEnabled' => false
         ];
+
+        $this->delayFunction = function (int $backoffCount) {
```

Comment:
why does this need to be closure?

Reply:
It's a closure so that we can test its accuracy in the test (see [this line](https://github.com/googleapis/google-cloud-php/pull/7664/files#diff-5921c1c17dd63bf234a77089aa092a7465eedaeb2a2d37eebfed4f93463a2e75R72)).

Reply:
I think it would be better to use the mock framework to create a spy for this method:
[prophesize(](https://github.com/phpspec/prophecy?tab=readme-ov-file#spies)

Reply:
AFAIK, It's not possible to do what you're asking with prophecy.